src/components/movieCard/movieCard.css

```diff
@@ -0,0 +1,55 @@
+.movie-card {
+    position: relative;
+    display: flex;
+    max-width: 100%;
+    margin-bottom: -4px;
+    z-index: 1;
+}
+
+.details {
+    position: absolute;
+    /* Absolute to position details above the card */
+    bottom: -6px;
+    left: 0;
+    right: 0;
+    padding: 20px;
+    /* Padding for readability */
+    z-index: 2;
+    display: none;
+}
+
+.movie-card:hover .details {
+    display: block;
+}
+
+.movie-card:hover .detail-image {
+    filter: brightness(40%);
+    /* Lowers the brightness for a dark hover effect */
+    transition: all 0.35s ease-in-out;
+    /* Makes the transition easier */
+    -webkit-filter: brightness(40%);
+    /* For browser compatibility */
+    -webkit-transition: all 0.35s ease-in-out;
+    /* For browser compatibility */
+}
+
+/* ------------ MEDIA QUERIES ------------ */
+@media (max-width: 569px) and (min-width: 369px) {
+    .movie-card {
+        max-width: 50%;
+    }
+}
+
+/* ------------ Tablet ------------ */
+@media (max-width: 819px) and (min-width: 570px) {
+    .movie-card {
+        max-width: 33%;
+    }
+}
+
+/* ------------ Desktop ------------ */
+@media (min-width: 820px) {
```

Comment:
I think the breakpoints are well chosen. Tips is to only use min-width as the max-width does not do much in this case :)